installation/kubernetes/helm/openclarity/templates/cr-discovery-server/daemonset.yaml

```diff
@@ -40,6 +40,9 @@ spec:
             - mountPath: /var/run/containerd
               name: containerd
               readOnly: true
+            - mountPath: /run/k3s/containerd
+              name: k3s-containerd
+              readOnly: true
```

Comment:
Sorry that I'm jumping right into this without ever contributed to this repository but I'm following this issue/problem aswell so wanted to contribute with my opinion on that topic:

Instead of mounting multiple paths it would be better if the mount path is configurable with a default value.
This will make sure that for future changes no release required. Also it will be more flexible between the systems where openclarity is being installed.

What do you think?

Reply:
It is indeed the best approach. It might introduce breaking changes however, unless we keep this list in Values.yaml.